### PR TITLE
docs(longhorn):  Add longhorn 1.2.6 images

### PR DESCRIPTION
**Pull request by weizhe0422**
Signed-off-by: Ray Chang <ray.chang@suse.com>

#### Pull Request Checklist ####

- [x] Change does not remove any existing Images or Tags in the images-list file
- [x] Change does not remove / overwrite exiting Images or Tags in Rancher DockerHub
- [x] If updating an existing entry, verify the `SOURCE` is still accurate and upstream hasn't been migrated to a new regitry or repo (if they've migrated, a new repo request to EIO is needed to comply with the `SOURCE DESTINATION TAG` pattern)
- [x] New entries are in format `SOURCE DESTINATION TAG`
- [x] New entries are added to the correct section of the list (sorted lexicographically)
- [x] New entries have a repo created in Rancher Dockerhub (where the image will be mirrored to)
- [x] New entries are licensed with Rancher favored licenses - Apache 2 and MIT - or approved licenses - as according to [CNCF approved licenses](https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md).
- [x] Changes to scripting or CI config have been tested to the best of your ability

#### Final Checks after the PR is merged ####
- [ ] Confirm that you can pull the new images and tags from DockerHub

cc @innobead @PhanLe1010

### CODE REVIEW

**Review by brandond — APPROVED** [review]

**Review by PhanLe1010 — APPROVED** [review]
LGTM

**Comment by PhanLe1010** [comment]
Can you help to merge in @brandond? Thank you

**Comment by khushboo-rancher** [comment]
Can we merge this? I can do a quick check once the images are merged.

**Pull request merged**
